### PR TITLE
fix: Bugs and CTF enhancements

### PR DESCRIPTION
**Pull request by iach526526**
- CTF request congestion->say wait to Discord
- sticker name error

## Summary by CodeRabbit

- **Bug Fixes**
  - Updated embed message in Discord bot to display the correct sticker (from 'logo' to 'zap').

- **Improvements**
  - Added deferral to bot commands to prevent timeout issues, ensuring smoother interactions.

- **Updates**
  - Changed announcement text in system status checks.
  - Modified gift sending functionality to use usernames instead of user IDs for improved user experience and error handling.

### CODE REVIEW

File: cog/ctf.py [hunk 1]
```diff
@@ -64,6 +64,7 @@ def __init__(self, *args, **kwargs) -> None:
 
                 async def callback(self, interaction: discord.Interaction):
                     try:
+                        await ctx.defer()  # 確保機器人請求不會超時
```
Comment (winstonsung) [inline cog/ctf.py:67]: Pylint
```
cog/ctf.py:67:30: E0602: Undefined variable 'ctx' (undefined-variable)
```

**Review by winstonsung — CHANGES_REQUESTED** [review]

**Review by Edit-Mr — APPROVED** [review]

**Comment by iach526526** [comment]
**Why Pylint always faild!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!**

**Comment by iach526526** [comment]
That is means lambda is forbidden?

**Comment by winstonsung** [comment]
https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/cell-var-from-loop.html

**Review by winstonsung — APPROVED** [review]

**Pull request merged**